packages/dev/test/RelayServerRequestsProfiling.test.ts

```diff
@@ -8,7 +8,7 @@ import { createServerLogger } from '@opengsn/logger/dist/ServerWinstonLogger'
 
 contract('RelayServerRequestsProfiling', function (accounts) {
   const refreshStateTimeoutBlocks = 2
-  const callsPerStateRefresh = 17
```

Comment:
please explain how comes there is one more RPC call after REMOVING a call to RPC api.